tools/build-fs.sh

```diff
@@ -112,11 +141,4 @@ echo "building full service..."
 # shellcheck disable=SC2086 # split away - Use BUILD_OPTIONS to set additional build options
 cargo build --release ${BUILD_OPTIONS}
 
-target_dir=${CARGO_TARGET_DIR:-"target"}
-
-if [[ "${target_dir}/release" != "${WORK_DIR}" ]]
-then
-    echo "  binaries are available in ${target_dir}/release and ${WORK_DIR}"
-    cp "${target_dir}/release/full-service" "${WORK_DIR}"
-    cp "${target_dir}/release/validator-service" "${WORK_DIR}"
-fi
```

Comment:
How come we're getting rid of this? It's been very useful to have to build script copy the built files directly into my ~/.mobilecoin/{NET} directory.

Reply:
This is to take up less space.  We haven't been building to that directory, but copying the binaries around. 

Here are some problems with the current setup

- This is to take up less space.  We haven't been building to that directory, but copying the binaries around. 
- Using $HOME/.mobilecoin  I can't work on multiple repo copies the the same time (which is how I keep multiple workflows straight), builds over write each other. 
- This isn't really compatible with mob prompt since $HOME is local to the container

Consolidating the files into CARGO_TARGET_DIR and making "WORK_DIR" a link allows us to have one standard path place that is relative to the repo we're working in.